journal.py

```diff
@@ -108,6 +123,12 @@ def all(cls, session=None):
             session = DBSession
         return session.query(cls).order_by(cls.created.desc()).all()
 
+    @classmethod
+    def get_article(cls, article_id, session=None):
+        if session is None:
+            session = DBSession
+        return session.query(cls).filter(cls.id == article_id).all()
```

Comment:
Okay, thanks for reminding me! I was going to go this route, but was in the middle of something and didn't want to bother changing my references then. I'll do this by the next round.